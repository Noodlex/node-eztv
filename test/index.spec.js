import EZTV from '..';

describe('EZTV', () => {
  it('should get shows', async () => {
    expect(await EZTV.getShows({ query: 'sherlock' })).toMatchSnapshot();
  });

  it('should get show episodes', async () => {
    expect(await EZTV.getShowEpisodes({ query: 'sherlock' })).toMatchSnapshot();
  });
});
