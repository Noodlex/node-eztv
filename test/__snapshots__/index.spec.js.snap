// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EZTV should get show episodes 1`] = `
Object {
  "episodes": Array [],
  "id": Object {
    "query": "sherlock",
  },
  "showTitle": "",
}
`;

exports[`EZTV should get shows 1`] = `
Array [
  Object {
    "id": 376,
    "slug": "sherlock",
    "status": undefined,
    "url": "/shows/376/sherlock/",
  },
]
`;
